### ISSUE TITLE
[css-color-5] Any identifier should be valid at parse time for RCS in `color()`

### ISSUE THREAD

**Issue by LeaVerou** [w3c/csswg-drafts#8412]
https://drafts.csswg.org/css-color-5/#relative-color-function

css-color-5 allows us to describe components of custom color spaces via the [`components`](https://drafts.csswg.org/css-color-5/#descdef-color-profile-components) descriptor. However, we cannot have syntax that becomes invalid at parse time based on the value of a descriptor.
Therefore, we should allow *any* identifier to be used for RCS in `color()` at parse time, and make it IACVT if the wrong identifiers are used.

cc @tabatkins @svgeesus

**Comment by tabatkins** [comment]
We should limit this behavior to only apply when a custom color space is used - it's useful to have invalid syntax when you use the wrong component names for a *known* color space.

But with that amendment, I agree.

Do we want IACVT, or just a default color (black)? Apologies, I don't recall off the top of my head whether we already have a consistent answer here in colors.

**Comment by svgeesus** [comment]
Yes, the components descriptor only applies to custom color spaces, not to the predefined ones.
